src/app/movies/movies.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <mat-form-field appearance="fill">
     <mat-label>Link filmu</mat-label>
-    <input matInput [(ngModel)]="movie.id">
+    <input matInput [(ngModel)]="movie">
```

Comment:
możesz to ewentualnie zapiąć do jakiegoś newMovieInput, movie to zbyt ogólna nazwa zmiennej w tej aplikacji, nic konkretnego nie mówi. To nie będzie film - to będzie stan inputu, to co user wprowadza